app/build.gradle

```diff
@@ -42,6 +42,13 @@ dependencies {
     implementation 'com.google.android.material:material:1.4.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.1'
 
+    // image loading library
+    implementation ("com.github.bumptech.glide:glide:4.11.0") {
+        exclude group: "com.android.support"
```

Comment:
We use `androidx` which replaced `com.android.support` so we wouldn't need to do this. I posted the official GitHub link for Glide below

https://github.com/bumptech/glide